mne/gui/_coreg.py

```diff
@@ -126,7 +126,7 @@ def __init__(self, info_file, subject=None, subjects_dir=None,
                  head_shape_points=None, eeg_channels=None, orient_glyphs=None,
                  scale_by_distance=None, project_eeg=None, mark_inside=None,
                  sensor_opacity=None, trans=None, size=None, bgcolor=None,
-                 show=True, standalone=False, interaction='terrain',
+                 show=True, block=False, interaction='terrain',
```

Comment:
It's only a few lines to deprecate here:
```suggestion
                 show=True, block=False, interaction='terrain', *,
                 standalone=None,
```
Then later
```
if standalone is not None:
    warn('standalone is deprecated and will be removed in 1.1, '
         'use block instead')
    block = standalone
```
plus a `latest.inc` line, so let's do it.

Reply:
It looks like you did not separately commit the "then later" part of the suggestion above